go.sum

```diff
@@ -0,0 +1,291 @@
+cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
If you run `go mod tidy` this will remove any unused dependencies and clean up go.sum file. Do this once you have upgraded libraries to latest versions :)